lib/Sema/TypeCheckDecl.cpp

```diff
@@ -4821,6 +4845,12 @@ class DeclChecker : public DeclVisitor<DeclChecker> {
       return true;
     }
 
+    if (functionHasImplicitlyUnwrappedResult(FD)) {
+      auto &C = FD->getASTContext();
+      FD->getAttrs().add(
+          new (C) ImplicitlyUnwrappedOptionalAttr(/* implicit= */ false));
```

Comment:
Should be implicit -- its not written in the source

Reply:
The attribute ("!") is written in the source.

Or do you mean something else?

Reply:
I think it doesn't really matter for things that don't have a proper spelling, but Slava's right. This is implicit because the attribute itself can't be written (or printed).